Packages/Core/Runtime/Events/AtomEvent.cs

```diff
@@ -33,19 +40,17 @@ public class AtomEvent<T> : AtomEventBase
         [Tooltip("The number of old values (between 0-10) being replayed when someone subscribes to this Event.")]
         private int _replayBufferSize = 1;
 
-        private Queue<T> _replayBuffer = new Queue<T>();
+        /// <summary>
+        /// Queue used as a replay buffer.
+        /// </summary>
+        private readonly Queue<T> _replayBuffer = new Queue<T>();
```

Comment:
reminds me of: https://i.redd.it/iuy9fxt300811.png

is a docstring for the sake of docstring actually necessary? especially on a _private_ field? what have you won? it is of Type Queue and its name is replayBuffer, so all the comment says is "used as a"

Reply:
Very true, just "looked" odd to me where most other parts of the class had docstrings and that didn't. Happy to remove if you feel it's adding to cognitive load.